Gemfile

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-gem 'rspec'
-gem 'rubocop'
-gem 'sinatra'
 gem 'guard-rspec'
 gem 'rerun'
+gem 'rspec'
+gem 'rubocop'
 gem 'rubocop-rspec'
+gem 'sinatra'
```

Comment:
Probably don't need this gem until later on as we're not actually doing any front-end stuff yet.